src/common/java/net/minecraftforge/gradle/common/task/ArchiveChecksum.java

```diff
@@ -29,12 +29,8 @@
 import com.google.common.hash.Hashing;
 import com.google.common.hash.HashingInputStream;
 
-import java.io.BufferedWriter;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.PrintWriter;
+import java.io.*;
```

Comment:
You have heaps of * import collapses throughout here, we don't collapse imports to *, can you un collapses all these please?

Reply:
Existing imports in several other files contradict that claim, but in any case, done.